### PR TITLE
feat: adds upgrade plan translation

### PR DESCRIPTION
**Pull request by ashish-data**

### CODE REVIEW

File: src/lib/translations/en-GB.json [hunk 1]
```diff
@@ -701,5 +701,31 @@
...
+        "atlas-play-boost-plan-subTitle":"Starting from",
+        "upgrade-plan": "Upgrade Plan",
+        "current-plan": "Current Plan",
+        "recommended": "Recommended",
+        "faq-1": "What does public cloud offer?",
+        "faq-2": "What does the virtual private cloud offer?",
+        "faq-3": "Can I afford the Atlas cloud?",
+        "faq-4": "What are the benefits of subscribing to Atlas?",
+        "faq-5": "How do I subscribe and mange my subscription?",
+        "faq-6": "How do I subscribe and mange my subscription?",
```
Comment (MartynRobotham) [inline src/lib/translations/en-GB.json:727]: Should 5 and 6 be the same string?

Comment (ashish-data) [reply src/lib/translations/en-GB.json:727]: typo..removed duplicate keys

File: src/lib/translations/en-GB.json [hunk 2]
```diff
@@ -701,5 +701,31 @@
...
+        "atlas-play-plan-description":"A light plan that lets you manage your business to help your clients to grow.",
+        "atlas-play-boost-plan-subTitle":"Starting from",
+        "upgrade-plan": "Upgrade Plan",
+        "current-plan": "Current Plan",
+        "recommended": "Recommended",
+        "faq-1": "What does public cloud offer?",
+        "faq-2": "What does the virtual private cloud offer?",
+        "faq-3": "Can I afford the Atlas cloud?",
+        "faq-4": "What are the benefits of subscribing to Atlas?",
+        "faq-5": "How do I subscribe and mange my subscription?",
```
Comment (MartynRobotham) [inline src/lib/translations/en-GB.json:726]: Sorry again, should this be manage?

Comment (ashish-data) [reply src/lib/translations/en-GB.json:726]: fixed the typo

**Review by MartynRobotham — APPROVED** [review]

**Pull request merged**